src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -459,74 +390,61 @@ private void DeployCode(ExecutionEnvironment env, TransactionSubstate substate,
             return env;
         }
 
-        private bool ValidateDeltaGasLimit(ContractEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
+        private bool ValidateDeltaGasLimit(PublicEntry entry, ExecutionEnvironment env, ITxTracer txTracer)
         {
-            if (entry.GasLimit <= (ulong) (env.CurrentBlock.GasLimit - env.CurrentBlock.GasUsed))
-            {
-                return true;
-            }
-            
-            TraceLogInvalidTx(entry, $"BLOCK_GAS_LIMIT_EXCEEDED {entry.GasLimit.ToString()} > {env.CurrentBlock.GasLimit.ToString()} - {env.CurrentBlock.GasUsed.ToString()}");
+            if (entry.GasLimit <= (ulong) (env.CurrentBlock.GasLimit - env.CurrentBlock.GasUsed)) return true;
+
+            TraceLogInvalidTx(entry,
+                $"BLOCK_GAS_LIMIT_EXCEEDED {entry.GasLimit.ToString()} > {env.CurrentBlock.GasLimit.ToString()} - {env.CurrentBlock.GasUsed.ToString()}");
             QuickFail(entry, env, txTracer);
             return false;
         }
 
-        private bool ValidateIntrinsicGas(ContractEntry entry,
+        private bool ValidateIntrinsicGas(PublicEntry entry,
             ExecutionEnvironment env,
             ulong intrinsicGas,
             ITxTracer txTracer)
         {
             if (_logger.IsEnabled(LogEventLevel.Verbose))
-            {
                 _logger.Verbose("Intrinsic gas calculated for {entry}: {intrinsicGas}", entry, intrinsicGas);
-            }
 
-            if (entry.GasLimit >= intrinsicGas)
-            {
-                return true;
-            }
-            
-            TraceLogInvalidTx(entry, $"GAS_LIMIT_BELOW_INTRINSIC_GAS {entry.GasLimit.ToString()} < {intrinsicGas.ToString()}");
+            if (entry.GasLimit >= intrinsicGas) return true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)